script/Ciel/Day 9/09_01_CIEL09_9.txt

```diff
@@ -458,27 +470,27 @@ But that's wrong. Completely wrong.
 -- Page 793, Offset 24231.
 -- 不変では<古|おそ>すぎる。
 // *that* permanence, because he then goes on to explain what 'real' permanence is.
-That permanence is too outdated.
+That permanence is woefully outdated.
 }
 [sha:de891460fecb02d97fee589ad1c2e4daddfd52ab]{
 -- Page 793, Offset 24232.
 -- 知覚、知識を更新するのは、常に『その時代』の人間だ。
-The humans of the %{i}current age%{/i} will always be the ones who update our knowledge and perception.
+Those who push the boundaries of knowledge and perception will always be humans living in the present.
 }
 [sha:5a70ec5f431b519370662ebcf25565ab0cccc0fd]{
 -- Page 793, Offset 24233.
 -- 『古い時代』で全能である事に何の価値があるだろう？
-What value is there in omnipotence from a %{i}former age%{/i}?
+What value is there in omnipotence from a bygone age?
 }
 [sha:66023521bd5da0397b524fcf565d48ea1f0ed3c5]{
 -- Page 794, Offset 24234.
 -- 完全とは在るものではなく、
-Perfection is not something which already is,
+Perfection cannot be something that already exists.
 }
 [sha:c72a48b6b19f846c4513ec30445d4786fded1eaf]{
 -- Page 794, Offset 24235.
 -- 辿り着くべき場所でなくてはならない。
-it must be a place to reach.
+It must be a place we've yet to reach.
```

Comment:
These two lines are pretty important (scratch that, this whole page is top 10 most important scenes in the game)
I feel like including  'yet' here muddles things more than they should. The important point to Roa is simply that perfection is not something exists, but that it can be reached. That it is a place that IS TO BE (辿り着く**べき**) reached. And that it has to be like that. So we might as well cut 'already' as well.
>Perfection cannot be something that exists.
>It has to be a place for us to reach.